client/state/utils/actions.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Internal dependencies
+ */
+import { UTILS_UNIQUEID_INCREMENT } from 'state/action-types';
+
+export function uniqueId( prefix ) {
+	return ( dispatch, getState ) => {
```

Comment:
I don't understand how this is supposed to work. How are the `dispatch` and `getState` arguments getting passed into this function? I don't think they are currently, but I'm also not sure how this would be modified to work correctly.

This isn't quite an action creator... nor is it a thunk that returns an action creator, because an action is never returned and an action creator is not supposed to dispatch its own action (though a _bound action creator_ can). I wonder if it make sense to separate a pure action creator out of `uniqueId()`.